okhttp/src/main/java/com/backblaze/b2/client/okHttpClient/B2StorageOkHttpClientBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2017, Backblaze Inc. All Rights Reserved.
+ * License https://www.backblaze.com/using_b2_code.html
+ */
+package com.backblaze.b2.client.okHttpClient;
+
+import com.backblaze.b2.client.B2AccountAuthorizer;
+import com.backblaze.b2.client.B2AccountAuthorizerSimpleImpl;
+import com.backblaze.b2.client.B2ClientConfig;
+import com.backblaze.b2.client.B2DefaultRetryPolicy;
+import com.backblaze.b2.client.B2RetryPolicy;
+import com.backblaze.b2.client.B2Sdk;
+import com.backblaze.b2.client.B2StorageClient;
+import com.backblaze.b2.client.B2StorageClientImpl;
+import com.backblaze.b2.client.B2StorageClientWebifier;
+import com.backblaze.b2.client.B2StorageClientWebifierImpl;
+import com.backblaze.b2.client.credentialsSources.B2Credentials;
+import com.backblaze.b2.client.credentialsSources.B2CredentialsFromEnvironmentSource;
+import com.backblaze.b2.client.webApiClients.B2WebApiClient;
+
+import java.util.function.Supplier;
+
+public class B2StorageOkHttpClientBuilder {
+
+    private static final String DEFAULT_MASTER_URL = "https://api.backblazeb2.com/";
+
+    private final B2ClientConfig config;
+    private Supplier<B2RetryPolicy> retryPolicySupplier;
+
+    @SuppressWarnings("WeakerAccess")
+    public static B2StorageOkHttpClientBuilder builder(String userAgent)  {
+        final B2Credentials credentials = B2CredentialsFromEnvironmentSource.build().getCredentials();
+        return builder(credentials.getApplicationKeyId(), credentials.getApplicationKey(), userAgent );
+    }
+
+    @SuppressWarnings("WeakerAccess")
+    public static B2StorageOkHttpClientBuilder builder(B2ClientConfig config) {
+        return new B2StorageOkHttpClientBuilder(config);
+    }
+
+    @SuppressWarnings("WeakerAccess")
+    public static B2StorageOkHttpClientBuilder builder(String applicationKeyID, String applicationKey, String userAgent) {
```

Comment:
same comment as above about whether this signature is needed.

Reply:
same reply as above.